requests_oauthlib/oauth2_session.py

```diff
@@ -317,6 +333,7 @@ def request(self, method, url, data=None, headers=None, withhold_token=False,
         """Intercept all requests and add the OAuth 2 token if present."""
         if not is_secure_transport(url):
             raise InsecureTransportError()
+
```

Comment:
FYI... it was not "random" I was trying to make it more readable... Nevertheless the whitespace is removed.